src/main/java/net/sf/jabref/logic/auxparser/AuxParserResult.java

```diff
@@ -50,6 +51,14 @@ public int getCrossRefEntriesCount() {
         return crossRefEntriesCount;
     }
 
+    public void countCrossRefEntry() {
+        crossRefEntriesCount++;
+    }
+
+    public void countNestedAuxFile() {
```

Comment:
To long to type with one hand (in my bed)... :-) I can sort out tomorrow though.